terraform/env_types/gcp/gke/k8s_setup/variables.tf

```diff
@@ -103,47 +103,36 @@ variable "monitoring_namespace" {
   description = "clusterwide monitoring namespace"
```

Comment:
I think you should delete `github_org_name` variable

Reply:
Done